interface/.dockerignore

```diff
@@ -1,4 +1,3 @@
 bin
 build
-node_modules
-vendor
+adage/adage/config.py
```

Comment:
I think this one will just uselessly filter out a file called `interface/adage/adage/config.py`, which does not exist.